episodes/imaging-software.md

```diff
@@ -0,0 +1,486 @@
+```
+Rotate - Click and drag
+Pan - Shift + click and drag
+Zoom - Scroll in/out
+```
+
+### Roll dimensions ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/roll.svg
+){alt="A screenshot of Napari's roll dimensions button" height='25px'} 
+
+This switches the order of dimensions in the viewer. For example, let's switch 
+back to the 2D view for our cells image and press the roll dimensions button 
+multiple times. You'll see that it switches between different orthogonal views 
+(i.e. at 90 degrees to our starting view). Pressing it 3 times will bring us 
+back to the original orientation.
+
+![](fig/roll-dims.png){alt="Three screenshots of the cells image in napari, 
+with different dimension order"}
+
+### Transpose dimensions ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/transpose.svg
+){alt="A screenshot of Napari's transpose dimensions button" height='25px'}
+
+This button swaps the two currently displayed dimensions. Again trying this for 
+our cells image, we see that the image becomes flipped. Pressing the button 
+again brings us back to the original orientation.
+
+![](fig/transpose-dim.png){alt="Two screenshots of the cells image in napari, 
+with dimensions swapped"}
+
+### Grid ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/grid.svg
+){alt="A screenshot of Napari's grid button" height='25px'}
+
+This button displays all images in a grid. Using this for our cells image, we 
+see the nuclei (green) displayed next to the cell membranes (purple), rather 
+than on top of each other.
+
+### Home ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/home.svg
+){alt="A screenshot of Napari's home button" height='25px'}
+
+This button brings the canvas back to its default view. This is useful if you 
+have panned/zoomed to a specific region and want to quickly get back to an 
+overview of the full image.
+
+## Layer list
+
+Now that we've seen the main controls for the viewer, let's look at the layer 
+list. 'Layers' are how Napari displays multiple items together in the viewer. 
+For example, currently our layer list contains two items - 'nuclei' and 
+'membrane'. These are both `Image` layers and are displayed in order, with 
+the nuclei on top and membrane underneath.
+
+![](fig/layer-list.png){alt="A screenshot of Napari's layer list, showing two 
+image layers named 'nuclei' and 'membrane'"}
+
+We can show/hide each layer by clicking the eye icon on the left side of their 
+row. We can also rename them by double clicking on the row.
+
+We can change the order of layers by dragging and dropping items in the layer 
+list. For example, try dragging the membrane layer above the nuclei. You should 
+see the nuclei disappear from the viewer (as they are now hidden by the membrane 
+image on top).
+
+![](fig/layer-reordering.png){alt="A screenshot of Napari with the nuclei and 
+membrane layer swapped"}
```

Comment:
This raises an interesting point. The "black" in the membrane is opaque and thus blocking out the nuclei, while the black in the nuclei appear to be transparent to let the layer below through. This appears to be controlled by the blending feature. If I were to swap the layers and the blending, I get what appears to be the same image as before. Probably not for this lesson, but something we might want to make clear in a future one.